readme.md

```diff
@@ -58,7 +58,7 @@ $ yo office [arguments] [options]
 The following command line arguments are supported. If using the command line arguments, you must use them in the order cited below, or the generator will prompt you for the values.
 
 #### `projectType`
-Framework to use for the project. The supported project types include JQuery (`jquery`), Angular (`angular`), React (`react`) and Excel Custom Functions (`excel-functions`). You can also use Manifest Only (`manifest`) which will create only the `manifest.xml` for an Office Add-in.
+Framework to use for the project. The supported project types include JQuery (`taskpane`), Angular (`angular`), React (`react`) and Excel Custom Functions (`excel-functions`). You can also use Manifest Only (`manifest`) which will create only the `manifest.xml` for an Office Add-in.

```

Comment:
Should the start of the second sentence be revised/reworded so that it doesn't use the word `JQuery` ?  (i.e., the options described by the second sentence here should be consistent with the info presented in the Yo Office prompts)

Reply:
@TCourtneyOwen So what about the usage of "JQuery"? I don't think that word should be used.

Reply:
Would "Vanilla JS" be an accurate/appropriate way of describing that project type (instead of "JQuery")?

Reply:
I don't think so. I would change the text to:

+ Specify the project type to create. `taskpane` for a TaskPane add-in using HTML, `angular` for a TaskPane add-in using the Angular framework, `react` for a TaskPane add-in using the React framework, `excel-functions` for a Task Pane add-in with Excel Custom Functions, or `manifest` for just the manifest and related files needed for an Office Add-in.

Reply:
@TCourtneyOwen I pushed a commit to use a table to show the values.